src/components/[guild]/EditGuild/components/Events/Events.tsx

```diff
@@ -0,0 +1,23 @@
+import { SimpleGrid, Text } from "@chakra-ui/react"
+import EventInput from "./EventInput"
+
+const Events = () => (
+  <>
+    <Text colorScheme="gray">
+      Guild can auto-import your events from different platforms to show them in one
+      place.
+    </Text>
+    <SimpleGrid columns={2} gap={3}>
+      <EventInput name={"EVENTBRITE"} />
+      <EventInput name={"LUMA"} />
+
+      {/*supportedEventSources
```

Comment:
You can remove this comment ;) And then add back when we'll actually have more event sources